packages/cheqd/tests/cheqd-did-resolver.e2e.test.ts

```diff
@@ -75,7 +75,11 @@ describe('Cheqd DID resolver', () => {
     })
     expect(JsonTransformer.toJSON(resolveResult)).toMatchObject({
       didDocument: {
-        '@context': ['https://www.w3.org/ns/did/v1', 'https://w3id.org/security/suites/jws-2020/v1'],
+        '@context': [          
+          'https://w3id.org/did/v1',
```

Comment:
why do we now have two did contexts? I don't think this will work with a JSON ld processor?

Reply:
Any reason cheqd uses the other one? The spec seems to mostly refer to the one used by credo.

Did you try it in a json-ld resolver with both contexts, does it cause issues?

We probably need a way to avoid this?

Reply:
Hey @TimoGlastra it seems that there might be an inconsistency between the `context` that Credo populates, and the `context` that is in the DID Core and DID Resolution specs.

Across the latter, it seems that "https://www.w3.org/ns/did/v1" is the default. Maybe this is a change that is needed on the Credo side?

Reply:
Tagging @berendsliedrecht here as well

Reply:
Hey @TimoGlastra @berendsliedrecht I confirmed with Markus Sabadello which is the correct `context` and he said that the "[https://www.w3.org/ns/did/v1](https://www.w3.org/ns/did/v1)" URL that the cheqd SDK populates is correct. 

This means that the auto-population within Credo should be updated to reflect this URL.

In the meantime, is there a workaround @DaevMithran @sownak - can we override the Credo `context` population or look to update the default `context` without breaking anything?